accession-commons-test/src/main/java/uk/ac/ebi/ampt2d/test/testers/HistoryTester.java

```diff
@@ -66,23 +68,23 @@ public HistoryAccessionTester assertEvent(int i, Consumer<HistoryEvent<TestModel
     }
 
     public static Consumer<HistoryEvent<TestModel, String>> assertAccession(String accession) {
-        return event -> assertEquals(accession, event.getAccession());
+        return event -> Assert.assertEquals(accession, event.getAccession());
```

Comment:
Is this change really needed, given that the static import points to the method directly?

Reply:
Nope, I was probably fighting with some import.